docs/guides/testing.md

```diff
@@ -72,3 +72,13 @@ def my_favorite_flow():
 def test_my_favorite_task():
     assert my_favorite_task.fn() == 42
 ```
+
+!!! tip "Disable logger"
+    If your task makes use of a logger, you can disable the logger in order to avoid the `RuntimeError` raised from a missing flow context.
```

Comment:
```suggestion
    If your task makes uses a logger, you can disable the logger in order to avoid the `RuntimeError` raised from a missing flow context.
```